From 8697997e332e8e128a71d83f7be471ee78b5c18a Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Mon, 20 Feb 2017 10:49:46 +0100
Subject: [PATCH] deviceproviderfactory: fix empty class check

---
 gst/gstdeviceproviderfactory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/gstdeviceproviderfactory.c b/gst/gstdeviceproviderfactory.c
index 305e214..4835f99 100644
--- a/gst/gstdeviceproviderfactory.c
+++ b/gst/gstdeviceproviderfactory.c
@@ -498,7 +498,7 @@ gst_device_provider_factory_has_classesv (GstDeviceProviderFactory * factory,
     const gchar *found;
     guint len;
 
-    if (classes[0] == '\0')
+    if (classes[0][0] == '\0')
       continue;
 
     found = strstr (klass, classes[0]);
-- 
2.9.3

